packages/edit-site/src/components/header-edit-mode/document-actions/index.js

```diff
@@ -128,7 +128,11 @@ function TemplateDocumentActions( { className, onBack } ) {
 
 	return (
 		<BaseDocumentActions
-			className={ className }
+			className={
+				record.wp_pattern_sync_status !== 'unsynced'
+					? classnames( 'is-synced-entity', className )
+					: className
+			}
```

Comment:
Tiny code style nit (not a blocker, and feel free to ignore!): if we think there might be more classnames at any point, instead of a ternary it could be written as:

```suggestion
			className={ classnames( className, {
				'is-synced-entity':
					record.wp_pattern_sync_status !== 'unsynced',
			} ) }
```

Reply:
🤦 thanks @andrewserong , not sure what I was thinking at that point - your version is the way that `classnames` is supposed to be used of course! Have updated.

Reply:
Thanks for updating! 🙂